ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/tap-aggregation.j2

```diff
@@ -0,0 +1,44 @@
+{# eos - tap-aggregation #}
+{% if tap_aggregation is arista.avd.defined %}
+!
+tap aggregation
+{%     if tap_aggregation.mode.exclusive is arista.avd.defined %}
+{%         if tap_aggregation.mode.exclusive.enabled is arista.avd.defined(true) %}
+{%             set mode_cli = "mode exclusive" %}
+{%         endif %}
+{%         if tap_aggregation.mode.exclusive.profile is arista.avd.defined %}
+{%             set mode_cli = mode_cli ~ " profile " ~ tap_aggregation.mode.exclusive.profile %}
+{%         endif %}
+   {{ mode_cli }}
+{%         for interface in tap_aggregation.mode.exclusive.no_errdisable | arista.avd.natural_sort %}
+   mode exclusive no-errdisable {{ interface }}
+{%         endfor %}
+{%     endif %}
```

Comment:
@tgodaA I did it like this on purpose, because EOS allows sth like this for example:

```
sh run sec tap agg
tap aggregation
   mode exclusive no-errdisable Ethernet1/1
   mac timestamp header format 64-bit
   mac fcs append
   mac fcs-error correct
```

so you can basically configure no errdisable on ifs, while the mode is not set..

but you cannot set a profile while the mode is false, these commands seem bundled.

====

keeping it like it is would come in handy for people that switch between tap and non tap mode and want to keep global tap config (so basically just switch it off and on). dunno if that happens in real life though

what do you think?

Reply:
I'm starting to feel that I've written this while half sleeping and half riding a bike, while speech typing with siri. 

let me fix it and thanks for reviewing!